intl/msg_hash_es.h

```diff
@@ -3843,7 +3976,7 @@ MSG_HASH(
 	)
 MSG_HASH(
 	MSG_FRAMES,
-	"Frames"
+	"Cuadros"
```

Comment:
Please don't frames it's extensively used and linked with FPS

Reply:
Ok. Thanks for the clarification.